week-2-wide-perspective/w02e03-pagination-problem/src/middleware.js

```diff
@@ -0,0 +1,25 @@
+export const paginationMiddleware = function (req, res, next) {
+    const {skip, limit} = req.query
+    let computedSkip = 0
+    if (skip) {
+        computedSkip = Number(skip)
+    }
+    let computedLimit
+    if (limit) {
+            computedLimit = Number(limit)
+    }
+    if(computedSkip && computedLimit) {
+        res.query = {skip: computedSkip, limit: computedLimit}
+    }
+    next()
+    }
+
+
+export const getPaginationParamas = (query) => {
```

Comment:
Ten getter to ciekawy pomysł. 

Jednak tutaj fajnie by było przekazywać coś takiego jak `req` a nie `query`
Wtedy to sam `getter` widziałby  zjakiego pola dla `req` ma odczytać obiekt z paginacją.